docs/modules/ow.md

```diff
@@ -264,5 +265,49 @@ Writes multi bytes. If `power` is 1 then the wire is held high at the end for pa
 #### Returns
 `nil`
 
-####See also
+#### See also
 [ow.depower()](#owdepower)
+
+## ow.timings()
```

Comment:
Mabe call it settimings and I am not sure if we need a getter.

Reply:
timings or settimings, either way works. Don't see the use for a getter though, most likely this is a call you make only once during program lifetime.

Reply:
A getter would show the initial settings, but as they are documented that seems fine. 
I really  would like a getter be named as that though.

Reply:
I mean that it should be named settimings.

Reply:
> it should be named settimings

I generally agree. Unfortunately we have no cohesive naming strategy across our code base (in the early days nobody paid attention to that). Hence, we find snake_case, camelCase and nocase style in different modules. The least we can do is trying to be consistent _within_ a module. That would require the function to be called `set_timings` IMO.

Reply:
Agreed

Reply:
ow.set_timings it is, pushed!

Reply:
Thanks